daprdocs/content/en/reference/components-reference/supported-bindings/s3.md

```diff
@@ -75,8 +75,9 @@ When running the Dapr sidecar (daprd) with your application on EKS (AWS Kubernet
 
 ### Using with Minio
 
+<!--IGNORE_LINKS-->
 [Minio](https://min.io/) is a service that exposes local storage as S3-compatible block storage, and it's a popular alternative to S3 especially in development environments. You can use the S3 binding with Minio too, with some configuration tweaks:
```

Comment:
This links seems to work as of now. Can we please try to remove the ignore links and see if it passes?